Source/src_ios/PasswordTester.swift

```diff
@@ -62,9 +62,9 @@ public class PasswordTester {
     ///
     /// - Parameter pin: PIN to evaluate..
     /// - Returns: Result of the testing
-    public func testPin(_ pin: String) -> PinTestResult {
+    public func testPin(_ passcodePtr: UnsafePointer<Int8>) -> PinTestResult {
```

Comment:
We should also keep string version of API, just like on Android.

Reply:
I might be missing the point but I believe there is an implicit conversion of string to UnsafePointer when C-side code requires `const char *str` which is also our case. Thanks to that, I thought changes in the API should not bother our users. I am ok with implementing both, just trying to understand the situation.

Reply:
You're absolutely right. I completely forgot that this Swift's feature exist. This is then correct and we should only update the documentation to encourage app developers to pass the strings to the function. I'm pretty sure that this API will make nervous a lot of regular app developers :)